ClinicalTrial_SE_curation.Rmd

```diff
@@ -0,0 +1,320 @@
+---
+title: "Curation of Clinical trial Sumamrized Experiment objects"
+author: "Nikta Feizi, Christopher Eeles, Sisira Nair"
+date: "`r format(Sys.time(), '%a/%d/%b')`"
+output: html_document
+---
+#Please note:
+Parameters of "run_all" function are data specific and must be defined by looking at the metadata outside of the function. 
+Run "assay_data <- as.data.frame(exprs(rma.norm)) " to find "delim" which is the delimiter that separates the GSM IDs from the assay_data column names.   
+Pheno_data  column names must be used to assign column names to pre-defined columns (in "prior" vector). 
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+
+```{r}
+# configure download options for this session
+options(encoding = "UTF-8" , timeout = 1000)
+work_dir = setwd(paste0(getwd()))
+```
+
+```{r}
+# fetch Gencode v33 annotations from BHKLAB-Pachyderm/Annotations
+gencode_url <- "https://github.com/BHKLAB-Pachyderm/Annotations/raw/master/Gencode.v33.annotation.RData"
+gencode_file <- file.path(work_dir, "Ensembl.v99.annotation.RData")
+download.file(gencode_url, destfile=gencode_file)
+load("Ensembl.v99.annotation.RData")
+```
+
+
+```{r}
+#required libraries
+library(stringr)
+library(GEOquery)
+library(SummarizedExperiment)
+library(qs)
+library(affy)
+```
+
+```{r}
+geo_rma<- function (raw_tar, geo_path , brain_path){
+  
+  # Download CEL files
+  cel.file <- paste(work_dir, raw_tar, sep="/")
```

Comment:
Best Practices:

Always build file paths with `file.path` because it does not assume a file separator for your OS. Linux and macOS use "/", but Windows uses "\". `file.path` will read the file separator for the OS and is thus operating system agnostic.

Reply:
@NiktaFeizi

Reply:
Done